themes/digital.gov/layouts/events/card-event.html

```diff
@@ -15,8 +15,10 @@ <h3>
       <div class="summary">{{ .Params.summary | markdownify }}</div>
       {{- end -}}
 
-      {{- if .Params.host -}}
-      <div class="host">Hosted by {{ .Params.host | markdownify }}</div>
+      {{- if .Params.community_name -}}
+        <p class="host">Hosted by Digital.gov's {{ .Params.community_name }} Community</p>
+        {{- else -}}
```

Comment:
The indent here on the `{{- else -}}` is confusing, can we make it match the `if` above?

Reply:
Done